apack.json

```diff
@@ -6,6 +6,7 @@
   "requiredFeatureLevel": "apack.core:9",
   "featureLevel": 106,
```

Comment:
bump feature level to 107. Also need to do this for matter extensions and gosdk

Reply:
sure